cpp/memilio/io/epi_data.h

```diff
@@ -65,6 +65,58 @@ class StringDate : public Date
     }
 };
 
+/**
+ * Represents the entries of a confirmed cases data file, e.g., from RKI.
+ * Number of confirmed, recovered and deceased in a Germany on a specific date.
+ * ConfirmedCasesNoAgeEntry is a simplified version of ConfirmedCasesDataEntry without agegroups.
+ */
+class ConfirmedCasesNoAgeEntry
+{
+public:
+    double num_confirmed;
+    double num_recovered;
+    double num_deaths;
+    Date date;
+
+    template <class IOContext>
+    static IOResult<ConfirmedCasesNoAgeEntry> deserialize(IOContext& io)
+    {
+        auto obj           = io.expect_object("ConfirmedCasesNoAgeEntry");
+        auto num_confirmed = obj.expect_element("Confirmed", Tag<double>{});
+        auto num_recovered = obj.expect_element("Recovered", Tag<double>{});
+        auto num_deaths    = obj.expect_element("Deaths", Tag<double>{});
+        auto date          = obj.expect_element("Date", Tag<StringDate>{});
+        return apply(
+            io,
+            [](auto&& nc, auto&& nr, auto&& nd, auto&& d) {
+                return ConfirmedCasesNoAgeEntry{nc, nr, nd, d};
+            },
+            num_confirmed, num_recovered, num_deaths, date);
+    }
+};
+
+/**
+ * Deserialize a list of ConfirmedCasesNoAgeEntry from json.
+ * @param jsvalue  json value, must be an array of objects, objects must match ConfirmedCasesNoAgeEntry.
+ *  Exactly one entry per date should be provided, for example no entries per age group.
+ * @return list of ConfirmedCasesNoAgeEntry.
+ */
+inline IOResult<std::vector<ConfirmedCasesNoAgeEntry>> deserialize_confirmed_cases_noage(const Json::Value& jsvalue)
+{
+    return deserialize_json(jsvalue, Tag<std::vector<ConfirmedCasesNoAgeEntry>>{});
```

Comment:
I see that you did this as in the respective function for DiviEntry. Is there a reason why you are not doing this using BOOST_OUTCOME_TRY as in ConfirmedCasesDataEntry? (The same holds for read_confirmed_cases_noage)

Reply:
Yes, the derialize_json function used is different from the one used for ConfirmedCasesDataEntry. Here we provide 2 arguments, in the function for ConfirmedCasesDataEntry only one. Our function does this BOOST_OUTCOME_TRY thing on its own.